server/sonar-web/src/main/js/apps/background-tasks/components/ScannerContext.js

```diff
@@ -56,7 +60,7 @@ export default class ScannerContext extends React.PureComponent {
     });
   }
 
-  handleCloseClick = (event: Event) => {
+  handleCloseClick = (event /*: Event */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)